### PR TITLE
Update vipps-login-api.md

### PR DESCRIPTION
**Pull request by VidarHolm**
Updated text to reflect improved mobile flow and recommendations for implementation from apps.

### CODE REVIEW

File: vipps-login-api.md [hunk 1]
```diff
@@ -86,17 +85,13 @@ The user goes to the Vipps app and confirms the login:
...
-This is the fallback flow for mobile devices if it is not possible to use the app-switch based flow. This is the same flow as for desktop above but please note that the user in this flow need to manually navigate back to the browser where they entered the phone number. If the user is remembered in browser the confirmation in app flow will be skipped.
 
-![Phone number based flow - mobile](images/Mobile_flow_with_phone_number.png)
+#### Mobile flow - app switch based flow
+If the user is on a moobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps-app if the user is not rememberd in browsere. The user will not be prompted to enter the phone number. In the Vipps-app the user confirms the login and can choose whether to be rememberd in the browser for later logins. After confirming in app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems. On the Vipps landing page the user will finalise the autentication and provide consents if required. Then the user is redirected back to the redirect URI provided by merchant (can be webpage or app).
```
Comment (jonkjenn) [inline vipps-login-api.md:90]:
```suggestion
If the user is on a mobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps-app if the user is not rememberd in browsere. The user will not be prompted to enter the phone number. In the Vipps-app the user confirms the login and can choose whether to be rememberd in the browser for later logins. After confirming in app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems. On the Vipps landing page the user will finalise the autentication and provide consents if required. Then the user is redirected back to the redirect URI provided by merchant (can be webpage or app).
```

Comment (jonkjenn) [inline vipps-login-api.md:90]:
```suggestion
If the user is on a moobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps-app if the user is not rememberd in browsere. The user will not be prompted to enter the phone number. In the Vipps-app the user confirms the login and can choose whether to be remembered in the browser for later logins. After confirming in app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems. On the Vipps landing page the user will finalise the autentication and provide consents if required. Then the user is redirected back to the redirect URI provided by merchant (can be webpage or app).
```

Comment (jonkjenn) [inline vipps-login-api.md:90]:
```suggestion
If the user is on a moobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps-app if the user is not remembered in browsere. The user will not be prompted to enter the phone number. In the Vipps-app the user confirms the login and can choose whether to be rememberd in the browser for later logins. After confirming in app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems. On the Vipps landing page the user will finalise the autentication and provide consents if required. Then the user is redirected back to the redirect URI provided by merchant (can be webpage or app).
```

Comment (jonkjenn) [inline vipps-login-api.md:90]:
```suggestion
If the user is on a moobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps-app if the user is not rememberd in browsere. The user will not be prompted to enter the phone number. In the Vipps-app the user confirms the login and can choose whether to be rememberd in the browser for later logins. After confirming in the app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems. On the Vipps landing page the user will finalise the autentication and provide consents if required. Then the user is redirected back to the redirect URI provided by merchant (can be webpage or app).
```

Comment (jonkjenn) [reply vipps-login-api.md:90]: ` After confirming in app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems.`

Synes denne var veldig tung, hva med noe ala
 After confirming in app the user needs to switch back to the Vipps page in the browser/app.

Comment (jonkjenn) [inline vipps-login-api.md:90]:
```suggestion
If the user is on a moobile device, the Vipps landing page in the browser will automatically trigger an app switch to the Vipps-app if the user is not rememberd in browsere. The user will not be prompted to enter the phone number. In the Vipps-app the user confirms the login and can choose whether to be rememberd in the browser for later logins. After confirming in app the user needs to switch back to the Vipps page in the initiating browser/app using the built in features in the operating systems. On the Vipps landing page the user will finalise the autentication and provide consents if required. The user is then redirected back to the redirect URI provided by the merchant (could be a webpage or an app).
```

File: vipps-login-api.md [hunk 2]
```diff
@@ -86,17 +85,13 @@ The user goes to the Vipps app and confirms the login:
...
 
-#### App-switch based flow - browser on mobile
-When the user starts to login with Vipps the browser automatically redirects the user to the Vipps-app, where login is confirmed and the user can choose not to be remembered in browser. After confirming in app the user is automatically redirected back to browser to finalise the autentication and provide consents if required. Then the user is redirected back to the redirect URI provided by merchant (can be webpage or app).
+![Mobile flow with app-switch](images/Mobile_flow_with_partial-app_switch.png)
 
-![Mobile flow with app-switch](images/Mobile_flow_with_app-switch.png)
-
-The app-switch based flow is only supported when the Vipps login API is called in a browser (on iOS only Safari is supported). You can open the API in the browser from an app, but webview in app is not supported.  
+We recommend that apps initiate Vipps login in a webview. SafariViewController and Chrome Custom Tabs is prefered as these webviews are able to utilise cookies stored in the user's browser.
```
Comment (jonkjenn) [inline vipps-login-api.md:94]:
```suggestion
We recommend that apps initiate Vipps login in a webview. SafariViewController or Chrome Custom Tabs are preferred as these webviews are able to utilize cookies stored in the user's browser.
```

File: vipps-login-api.md (hunk 1 above)
Comment (Oyvinkg) [reply vipps-login-api.md:90]: Typo på moobile

**Pull request merged**